frontend/src/scenes/actions/actionLogic.ts

```diff
@@ -35,11 +35,11 @@ export const actionLogic = kea<actionLogicType>([
                 },
             },
         ],
-        matchingPluginConfigs: [
-            null as PluginConfigWithPluginInfoNew[] | null,
+        matchingHogFunctions: [
+            null as HogFunctionType[] | null,
             {
-                loadMatchingPluginConfigs: async () => {
-                    const res = await api.actions.listMatchingPluginConfigs(props.id!)
+                loadMatchingHogFunctions: async () => {
+                    const res = await api.hogFunctions.list({ filters: { actions: [{ id: `${props.id}` }] } })
```

Comment:
There's some magic making this work. Will it only work if the action we're interested in is the first one, or will it match all?

Reply:
Added a test - it actually does work!